CMakeLists.txt

```diff
@@ -115,9 +120,10 @@ if(PROJECT_IS_TOP_LEVEL)
     src/*.h src/*.cc
     benchmark/*.h benchmark/*.cc
     test/*.h test/*.cc)
-  sourcemeta_target_clang_tidy(SOURCES src/*.cc)
 endif()
 
+sourcemeta_disable_clang_tidy()
```

Comment:
Interesting. Disabling looks a bit weird though. Now that I see this I wonder if it is possible to set `CMAKE_CXX_CLANG_TIDY` on a per-target level. If so, we could have a `CLANG_TIDY` option when declaring a library with `sourcemeta_library` (or executable with `sourcemeta_executable`). Then we don't have to worry about unsetting it, as we literally opt into it for specific things we are about?